src/Components/DesignAdjustments.tsx

```diff
@@ -10,25 +10,21 @@ export const DesignAdjustments = connect(
     const styles: string[] = []
 
     const primary = root.get('siteColorPrimary')
-    if (primary) styles.push(`--bs-primary: ${primary};`)
-
-    const primaryLighten = root.get('siteColorPrimaryLighten')
-    if (primaryLighten) styles.push(`--bs-primary-lighten: ${primaryLighten};`)
-
-    const primaryDarken = root.get('siteColorPrimaryDarken')
-    if (primaryDarken) styles.push(`--bs-primary-darken: ${primaryDarken};`)
-
-    const secondary = root.get('siteColorSecondary')
-    if (secondary) styles.push(`--bs-secondary: ${secondary};`)
-
-    const secondaryLighten = root.get('siteColorSecondaryLighten')
-    if (secondaryLighten) {
-      styles.push(`--bs-secondary-lighten: ${secondaryLighten};`)
+    if (primary) {
+      styles.push(`--bs-primary: ${primary};`)
+      styles.push(`--bs-primary-lighten: hsl(from ${primary} h s calc(l + 8));`)
+      styles.push(`--bs-primary-darken: hsl(from ${primary} h s calc(l - 8));`)
```

Comment:
* [ ] We should not duplicate the CSS code here. The current reason just using the CSS rules don't work is that the CSS applies to `:root` while the styles here only apply to `body` (see line 40). I suggest to apply the styles to `:root`. E.g.:
```tsx
  <Helmet style={[{ cssText: `:root{${styles.join('')}}` }]} />
```